git_machete/contexts.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, List
+
+
+class CommandLineContext:
```

Comment:
Hmmm this class name doesn't clearly indicate what this class does... although that's probably gonna be massively reworked soon anyway within the scope of https://github.com/VirtusLab/git-machete/issues/158 and https://github.com/VirtusLab/git-machete/issues/161

Reply:
Anywa, I'd just rename it to `CommandLineOptions`